src/rest/v0/block/next_id/mod.rs

```diff
@@ -0,0 +1,62 @@
+use crate::rest::server_service::PathPredicate;
+use actix_web::error::{Error as ActixError, ErrorBadRequest, ErrorInternalServerError};
+use actix_web::{App, Path, Query, State};
+use blockcfg::mock::Mockchain;
+use blockchain::BlockchainR;
+use bytes::Bytes;
+use chain_core::property::Settings;
+use chain_storage::store::BlockStore;
+
+pub fn create_handler(
+    blockchain: BlockchainR<Mockchain>,
+) -> impl Fn(&str) -> App<BlockchainR<Mockchain>> + Send + Sync + Clone + 'static {
+    move |prefix: &str| {
+        let full_prefix = format!("{}/v0/block", prefix);
+        App::with_state(blockchain.clone())
+            .filter(PathPredicate::new(&full_prefix, "[^/]+/next_id"))
```

Comment:
Same here, you might slip in a backslash and it would get interpreted as a string literal escape.